docs/recipes/hyperlinks.md

```diff
@@ -0,0 +1,78 @@
+# Use hyperlinks directly in the terminal
+
+This recipe is an example on how to leverage hyperlinks directly in the terminal.
+Hyperlinks are generated by terminal applications by using the [OSC-8 escape sequences](https://gist.github.com/egmontkob/eb114294efbcd5adb1944c9f3cb5feda).
+
+## Requirements
+
+By default, `ls` doesn't generate hyperlinks, here is an example of aliases to enable them in some common tools:
+
+```sh
+alias ls='ls --hyperlink --color=auto'
```

Comment:
while this may beyond the initial scope of this recipe, it may be worthwhile to note that core `ls` is not the only `ls` tool which provides hyperlink support and could provide an alternative to the coreutils on MacOS if desired.

this is actually what I use and afaik maintains argument compatibility as well as additional theming and other features.

[eza-community/eza](https://github.com/eza-community/eza?tab=readme-ov-file)